include/af/image.h

```diff
@@ -712,6 +712,27 @@ AFAPI array canny(const array& in, const cannyThreshold thresholdType,
                   const float lowThresholdRatio, const float highThresholdRatio,
                   const unsigned sobelWindow = 3, const bool isFast = false);
 #endif
+
+#if AF_API_VERSION >= 36
+/**
+   C++ Interface for gradient anisotropic(non-linear diffusion) smoothing
+
+   \param[in] in is the input image, expects non-integral (float/double) typed af::array
+   \param[in] timestep is the time step used in solving the diffusion equation.
+   \param[in] conductance parameter controls the sensitivity of conductance in diffusion equation.
+   \param[in] iterations is the number of times the diffusion step is performed.
+   \param[in] fftype indicates whether quadratic or exponential flux function is used by algorithm.
+    \param[in] isCurvatureDiffusion will let the user do curvature based diffusion instead
+               of the default gradient based diffusion.
+   \return an af::array containing anisotropically smoothed image pixel values
+
+   \ingroup image_func_anisotropic_diffusion
+*/
+AFAPI array anisotropicDiffusion(const af::array& in, const float timestep,
+                                 const float conductance, const unsigned iterations,
+                                 const fluxFunction fftype=AF_FLUX_EXPONENTIAL,
+                                 const diffusionEq eq=AF_DIFFUSION_GRAD);
```

Comment:
`eq` is not documented. A better name would be nice here. Also it doesn't need to be const.